src/actions/run_actor.js

```diff
@@ -26,52 +26,32 @@ module.exports = async function runActor({ apifyClient, actorId, input, memory,
     // If no run ID, starts new one
     if (!runId) {
         // Actor run options
-        const opts = { actId: actorId };
+        let opts = { actId: actorId };
+        if (memory) opts.memory = parseInt(memory, 10);
+        if (build) opts.build = build;
+
         const inputFile = await getInputFile();
         if (inputFile) {
-            const keyValueStoresClient = apifyClient.keyValueStores;
-            const store = await keyValueStoresClient.getOrCreateStore({ storeName: NAME_OF_KEBOOLA_INPUTS_STORE });
-            const storeId = store.id;
-            const key = apifyHelper.randomHostLikeString();
-            await keyValueStoresClient.putRecord({
-                storeId,
-                key,
-                body: inputFile,
-                contentType: 'text/csv',
-            });
-            const inputTableRecord = { storeId, key };
-            if (input) {
-                Object.assign(input, { inputTableRecord });
-            } else {
-                input = { inputTableRecord };
-            }
+            const inputTableRecord = await apifyHelper.uploadInputTable(apifyClient, inputFile);
```

Comment:
I know this was there before but could you describe the flow a bit? What is the difference between input and input file etc..